pkg/console/subresource/deployment/deployment.go

```diff
@@ -178,11 +178,13 @@ func consoleContainer(cr *v1alpha1.Console) corev1.Container {
 		TerminationMessagePolicy: corev1.TerminationMessagePolicy("File"),
 		Resources: corev1.ResourceRequirements{
 			Limits: map[corev1.ResourceName]resource.Quantity{
-				// TODO: fill these out
-				//	"cpu": int64(100),
-				//	"memory": int64(100)
+				corev1.ResourceCPU:    resource.MustParse("100Mi"),
```

Comment:
Great this looks like what we need.  Now to decide if the values are good, and if they should match the operator or not.

Reply:
Was spreaking with @mfojtik about setting the values on the operator and he advised that setting the `requests` && `limits` values makes no sense on the operator.
Also having the `Limits` set on the console deployment, since if by any reason that limit is hit, the pod will be killed by kernel.
So lets make it simple and just set the `Requests` on the deployments podTemplate ?

Reply:
@spadgett thoughts ?

Reply:
I spoke to @derekwaynecarr, and he recommended we set only request for both the operator and console. Let's do that.